### DISCUSSION TITLE
Can we retrieve links from a site like O365 SharePoint sites that require authentification?

### DISCUSSION THREAD

**Discussion by Tak-0511** [stevenvachon/broken-link-checker#215] [General]
Thank you for a great tool.

I can retrieve links from non-authentification sites,
but I cannot retrieve links from O365 SharePoint sites that require authentification.
Is there any way to retrieve links from sites that require authentification?

**Environment:**
 - OS and version: Windows 10
 - broken-link-checker version:12.13.0 LTS

Best Regards,
Tak

**Comment by Tak-0511** [comment]
Input : blc https:....com -ro
Output :Getting links from: https:....com
             Error: HTML could not be retrieved

**Comment by Cloudmaister** [comment]
Have you ever found a tool to check O365 Links?  Im still looking for one too.
